extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -1752,19 +1752,20 @@ describes.realWin('Events', {amp: 1}, env => {
         )
         .returns(unlisten)
         .once();
-      tracker.add(analyticsElement, 'visible', {}, eventResolver);
-      yield macroTask();
+      const res = tracker.add(analyticsElement, 'visible', {}, eventResolver);
+      expect(res).to.be.a('function');
+      await macroTask();
       saveCallback.callback({totalVisibleTime: 10});
-      return eventPromise.then(function*(event) {
-        expect(event.target).to.equal(root.getRootElement());
-        expect(event.type).to.equal('visible');
-        expect(event.vars.totalVisibleTime).to.equal(10);
-        yield macroTask();
-        expect(unlisten).to.be.called;
-      });
+      const event = await eventPromise;
+      expect(event.target).to.equal(root.getRootElement());
+      expect(event.type).to.equal('visible');
+      expect(event.vars.totalVisibleTime).to.equal(10);
+      expect(unlisten).to.be.not.called;
+      await res();
```

Comment:
The test checks if the unlistener from `tracker.add()` is called afterward but invoking `res()` calls the unlistener manually. Please fix

Reply:
As discussed offline, we will remove the check for `unlisten` (because it doesn't have too much value)